rust/utils/run-locally/src/cosmos/deploy.rs

```diff
@@ -53,16 +53,26 @@ pub fn deploy_cw_hyperlane(
     );
 
     // deploy igp set
+    #[cw_serde]
+    pub struct Something {
```

Comment:
I assume this should be named `InstantiateMsg`

Reply:
Yes, exactly. It's quite weird that same payload does not working

Reply:
I'd like to avoid generic names like `Something` though, could you use a more descriptive name?